cmd/main.go

```diff
@@ -185,12 +185,19 @@ func newTracerProvider(cfg config.SSIServiceConfig) (*sdktrace.TracerProvider, e
 		sdktrace.WithResource(resource.NewWithAttributes(
 			semconv.SchemaURL,
 			semconv.ServiceNameKey.String(config.ServiceName),
-			semconv.ServiceVersionKey.String(cfg.Version.SVN),
+			semconv.ServiceVersionKey.String(cfg.Svn),
 		)),
 	)
 	return tp, nil
 }
 
 func createLogFile(location string) string {
+	_, err := os.Stat(location)
+	if os.IsNotExist(err) {
+		err = os.MkdirAll(location, 0766)
+		if err != nil {
+			panic(err)
+		}
```

Comment:
two things - can combine the err check into one line and let's not panic here and instead return an err

Reply:
ok. i will fix it.